src/background/services/openPayments.ts

```diff
@@ -112,13 +112,15 @@ const enum ErrorCode {
   KEY_ADD_FAILED = 'key_add_failed',
 }
 
-const enum GrantResult {
-  SUCCESS = 'grant_success',
-  ERROR = 'grant_error',
+const enum Result {
+  GRANT_SUCCESS = 'grant_success',
+  GRANT_ERROR = 'grant_error',
+  KEY_ADDED = 'key_added',
 }
 
 const enum InteractionIntent {
```

Comment:
@sidvishnoi can we get a list with the missing intents/results for the welcome page? IIRC we discussed some of them before the summit, but I cannot recall which ones.

Reply:
results:
- `key_add_failed` or `key_add_error`
- `key_added` or `key_add_success` (this one)

intents:
- connect
- add_funds
- reconnect (this one)
- update_budget (missing from earlier)